Userland/Libraries/LibWeb/SVG/SVGScriptElement.cpp

```diff
@@ -19,4 +20,40 @@ void SVGScriptElement::initialize(JS::Realm& realm)
     set_prototype(&Bindings::ensure_web_prototype<Bindings::SVGScriptElementPrototype>(realm, "SVGScriptElement"));
 }
 
+void SVGScriptElement::visit_edges(Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    visitor.visit(m_script);
+}
+
+void SVGScriptElement::process_the_script_element()
```

Comment:
I believe we should include the [spec URL](https://www.w3.org/TR/SVGMobile12/script.html#ScriptContentProcessing) associated to the spec steps mentioned below

Reply:
yup, added it.